sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java

```diff
@@ -39,6 +44,10 @@ CosmosDiagnostics clientSideRequestStatistics(ClientSideRequestStatistics client
      */
     @Override
     public String toString() {
+        if (this.feedResponseDiagnostics != null) {
+            return feedResponseDiagnostics.toString();
```

Comment:
Do we not have client side request statistics for query operations ?

Reply:
We should also move FeedResponseDiagnostics to implementation package.

Reply:
> Do we not have client side request statistics for query operations ?

It is inside QueryMetrics type